### ISSUE TITLE
multi-returns don't carry over function calls

### ISSUE THREAD

**Issue by RicardoLuis0** [ZDoom/gzdoom#2210]
multi-returns don't carry over any function calls, ex.:
```
double, double ret1() { return 10, 20; }

double, double ret2() { return ret1(); }
````
ret2 will only return 10, not 20

**Comment by Player701** [comment]
FYI, this issue is very old and has already been reported before multiple times on the forums, e.g. [here](https://forum.zdoom.org/viewtopic.php?t=62813) and [here](https://forum.zdoom.org/viewtopic.php?p=1084443).

**Comment by RicardoLuis0** [comment]
yeah, i'm just putting it here and assigning to myself so that i don't forget to fix it
